kong/clustering/wrpc_data_plane.lua

```diff
@@ -219,7 +219,7 @@ function _M:communicate(premature)
       if hash == true then
         hash = DECLARATIVE_EMPTY_CONFIG_HASH
       end
-      assert(peer:call("ConfigService.PingCP", { hash = hash }))
+      assert(peer:void_call("ConfigService.PingCP", { hash = hash }))
```

Comment:
`void_call` is a bit strange, could we change it to a better name?

Reply:
how about `call_no_return`?

Reply:
How about `async_call`?